### PR TITLE
[compose-latest] Update samples on compose-latest branch to Compose 1.3.0-beta01

### PR DESCRIPTION
**Pull request by simona-anomis**
Update samples on compose-latest branch to Compose 1.3.0-beta01

### CODE REVIEW

**Review by arriolac — APPROVED** [review]
LGTM

**Comment by bentrengrove** [comment]
I'll finish this off for Sim once Accompanist is ready

File: Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt [hunk 1]
```diff
@@ -148,8 +148,6 @@ fun ConversationContent(
                 channelMembers = uiState.channelMembers,
                 onNavIconPressed = onNavIconPressed,
                 scrollBehavior = scrollBehavior,
-                // Use statusBarsPadding() to move the app bar content below the status bar
-                modifier = Modifier.statusBarsPadding(),
```
Comment (alexvanyo) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt:152]: 🎉

Comment (bentrengrove) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt:152]: Handled automatically by TopAppBar

File: Jetchat/app/src/main/java/com/example/compose/jetchat/profile/ProfileFragment.kt [hunk 2]
```diff
@@ -21,6 +21,7 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.compose.foundation.background
```
Comment (ricknout) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/profile/ProfileFragment.kt:24]: Are these new imports needed?

**Pull request merged**